lgc/builder/InOutBuilder.cpp

```diff
@@ -105,6 +105,7 @@ Value *InOutBuilder::CreateReadPerVertexInput(Type *resultTy, unsigned location,
   Value *even = nullptr;
 
   auto primType = getPipelineState()->getPrimitiveType();
+  auto provokingMode = getPipelineState()->getRasterizerState().provokingVertexMode;
```

Comment:
please use same variable name in whole change. (provokingMode -> provokingVertexMode)

Reply:
Fixed, thanks.